app/backend/pyproject.toml

```diff
@@ -111,7 +111,7 @@ module = ["http_ece", "py_vapid", "luhn", "sqlalchemy_utils.*"]
 ignore_missing_imports = true
 
 [tool.coverage.run]
-omit = ["src/couchers/proto/*", "src/couchers/migrations/*", "src/dummy_data.py"]
+omit = ["src/couchers/proto/**", "src/couchers/migrations/versions/**", "src/tests/**", "src/dummy_data.py"]
```

Comment:
Here's an argument for measuring coverate of our tests from coverage.py author: https://nedbatchelder.com/blog/202008/you_should_include_your_tests_in_coverage.html

Reply:
I think right now no one is really looking at the coverage. I don't look at it partly because it's too noisy.

I'm down to do this either way, not strongly opinionated. But we do need to exclude protos and migrations just make it messy.

Reply:
Let's keep the tests, and omit protos and migrations then?